sdk/program/src/entrypoint.rs

```diff
@@ -135,6 +141,7 @@ pub const MAX_PERMITTED_DATA_INCREASE: usize = 1_024 * 10;
 ///
 /// # Safety
 #[allow(clippy::type_complexity)]
+#[allow(clippy::integer_arithmetic)] // Jack May says it's safe :)
```

Comment:
@jackcmay please sign and initial the dotted line acknowledging that you are being thrown under a bus :slightly_smiling_face:

Reply:
As long as these are applied to the buffer serialized by the runtime it it safe, if used for anything else then use at your own rist